pkg/util/config/config.go

```diff
@@ -83,13 +83,28 @@ type LogicalBackup struct {
 	LogicalBackupS3SSE             string `name:"logical_backup_s3_sse" default:"AES256"`
 }
 
+// Operator options for connection pooler
+type ConnectionPool struct {
+	NumberOfInstances            *int32 `name:"connection_pool_number_of_instances" default:"1"`
+	Schema                       string `name:"connection_pool_schema" default:"pooler"`
+	User                         string `name:"connection_pool_user" default:"pooler"`
+	Image                        string `name:"connection_pool_image" default:"registry.opensource.zalan.do/acid/pgbouncer:master-5"`
+	Mode                         string `name:"connection_pool_mode" default:"transaction"`
+	MaxDBConnections             *int32 `name:"connection_pool_max_db_connections" default:"60"`
```

Comment:
What if someone configures max connection to be below 60? Should we really define a default here? Maybe this parameter should rather define a ratio to configured max_connections instead, e.g. between 0 and 1, where 1 is all connections minus reserved ones. And default would be 0.6 etc.

Reply:
60 is actually more like `last level defaults`, which would be almost always overridden in the operator/db cluster configuration. But I believe it's better to have an independent limit here, since max db connections depends mostly on how many resources are available, and max client connections (on the target database side, not on the pooler) depends mostly on estimations how many clients could be there.